README.md

```diff
@@ -245,6 +245,8 @@ For more details of advanced configuration, please see [Uniffle Coordinator Guid
 |rss.storage.type|-|Supports MEMORY_LOCALFILE, MEMORY_HDFS, MEMORY_LOCALFILE_HDFS|
 |rss.server.flush.cold.storage.threshold.size|64M| The threshold of data size for LOACALFILE and HDFS if MEMORY_LOCALFILE_HDFS is used|
 |rss.server.tags|-|The comma-separated list of tags to indicate the shuffle server's attributes. It will be used as the assignment basis for the coordinator|
+|rss.server.single.buffer.flush.enabled|false|Whether single buffer flush when size exceeded rss.server.single.buffer.flush.enabled|
```

Comment:
Should we change 
from `exceeded rss.server.single.buffer.flush.enabled`
to `rss.server.single.buffer.flush.threshold`?

Reply:
my mistake